incubator/hnc/test/conversion/conversion_test.go

```diff
@@ -584,6 +589,9 @@ func verifyConversion() {
 	checkCRDVersionInField("v1alpha2", ".spec.versions", true)
 	checkCRDVersionInField("v1alpha1", ".status.storedVersions", false)
 	checkCRDVersionInField("v1alpha2", ".status.storedVersions", true)
+
+	// Make sure HNC is up and running again
+	ensureVWHReady()
```

Comment:
Is this not needed but just to be safer?

Reply:
With flaky tests it's hard to tell. After I added this, the amount of flakiness did seem to go down considerably.